src/routes/index.js

```diff
@@ -1,8 +1,9 @@
-const express = require("express");
-const router = express.Router();
+const router = require("express").Router();
+const faculties = require("./check-faculties");
```

Comment:
since the endpoint is /faculties, what do you think about renaming the file "check-faculties" to "faculties" as well?